locale/pl/LC_MESSAGES/client.po

```diff
@@ -412,6 +412,8 @@ msgid ""
 "<a href=\"https://support.mozilla.org/kb/im-having-problems-with-my-firefox-"
 "account\">Help</a>"
 msgstr ""
+"<a href=\"https://support.mozilla.org/pl/kb/rozwiazywanie-problemow-z-kontem-"
```

Comment:
```
>> UNEXPECTED ATTRIBUTE VALUE: (locale/pl/LC_MESSAGES/client.po): href=https://support.mozilla.org/pl/kb/rozwiazywanie-problemow-z-kontem-firefoksa [<a href="https://support.mozilla.org/pl/kb/rozwiazywanie-problemow-z-kontem-firefoksa">Pomoc</a>]
```

Why is this value being changed? SUMO should be able to detect language automatically